assets/lokomotive-kubernetes/bootkube/assets.tf

```diff
@@ -68,7 +68,7 @@ resource "template_dir" "kubernetes" {
 # Populate kubernetes chart values file named kubernetes.yaml.
```

Comment:
I'd change commit message to: `assets/lokomotive-kubernetes/bootkube: format using terraform fmt`

Reply:
Commit message length is greater then 50 characters